src/server/mod.rs

```diff
@@ -189,6 +187,42 @@ fn parse_semantic_graph(
     }
 }
 
+enum LspServerCommand {
+    InjectMeasurement,
+    InjectTag,
+    InjectTagValue,
+}
```

Comment:
Injecting a field, and a field value, would be handled different. (The AND versus OR, or a different flux stdlib func.)

Reply:
Also...where is the injectFunction?